index.html

```diff
@@ -0,0 +1,206 @@
+      </div>
+      <div class="trial">
+        <p class='p1'></p>
+        <p class='p2'></p>
+      </div>
+    </section>
+    <section class='main-program' id='main-program'>
+      <h3 class='main-program-title'>Main Program </h3>
+      <h3 class='indicator'></h3>
+      <ul class='program-list'>
+        <li class="program">
+          <a href=""><img src="images/lecture.png" alt="Degree" class='degree-icon'></a>
+          <h4 class='program-name'>Degrees</h4>
+          <p class='program-description'>Lorem ipsum dolor sit amet consectetur adipisicing elit. Soluta itaque corrupti rerum fugit. Ipsam ipsa.</p>
+        </li>
+        <li class="program">
+          <a href=""><img src="images/exhibition.png" alt="Degree" class='degree-icon'></a>
+          <h4 class='program-name'>Certificates</h4>
+          <p class='program-description'>Lorem ipsum dolor sit amet consectetur adipisicing elit. Soluta itaque corrupti rerum fugit. Ipsam ipsa.</p>
+        </li>
+        <li class="program">
+          <a href=""><img src="images/forum.png" alt="Degree" class='degree-icon'></a>
+          <h4 class='program-name'>Short Trainings</h4>
+          <p class='program-description'>Lorem ipsum dolor sit amet consectetur adipisicing elit. Soluta itaque corrupti rerum fugit. Ipsam ipsa.</p>
+        </li>
+        <li class="program">
+          <a href=""><img src="images/workshop.png" alt="Degree" class='degree-icon'></a>
+          <h4 class='program-name'>Free Courses</h4>
+          <p class='program-description'>Lorem ipsum dolor sit amet consectetur adipisicing elit. Soluta itaque corrupti rerum fugit. Ipsam ipsa.</p>
+        </li>
+        <li class="program">
+          <a href=""><img src="images/ignite.png" alt="Degree" class='degree-icon'></a>
+          <h4 class='program-name'>Tutorials</h4>
+          <p class='program-description'>Lorem ipsum dolor sit amet consectetur adipisicing elit. Soluta itaque corrupti rerum fugit. Ipsam ipsa.
+          </p>
+        </li>
+      </ul>
+      <button class='join-btn'>Enroll Now</button>
+    </section>
+    <section id='instructor-section'>
+      <div class='instructor' id='instructor'>
+      <h3 class='instructor-title'> Course Instructors</h3>
+     <div class="instructor-card1" id='instructor-card1'>
+        <div class='photo-container'>
+          <img src="images/eyob.jpg" alt="instructor photo" class='instructor-photo'>
+        </div>
+        <div class='instructor-profile'>
+            <h4 class='instructor-name'>Eyob Temesgen</h4>
+            <span class='job'>Graduate of Software Security from DBU
+            </span>
+          <p class='Instructor-info'>Lorem ipsum, dolor sit amet consectetur adipisicing elit.</p>
+        </div>  
+      </div>
+      <div class="instructor-card2" id='instructor-card2'>
+        <div class='photo-container'>
+          <a href=""><img src="images/Ag.jpg" alt="instructor photo" class='instructor-photo'></a>
+        </div>
```

Comment:
- [ ] it is required that the featured section should be rendered dynamically using Javascript.